constantine/math/constants/banderwagon_sqrt.nim

```diff
@@ -17,3 +18,1344 @@ const
+      Fp[Banderwagon].fromHex"0x697431feceffb21884061f9f18bf5ab67d5d62341f399e527a79183fcde843c9",
+      Fp[Banderwagon].fromHex"0x2f00a2325ba21faac6e50c813baa9974308f60f019fd400926be16cb704b990a",
+      Fp[Banderwagon].fromHex"0x36294ce5aa3941ee054ba2513a8f6466a4170673bd1b99a3d922fa1a3b513f68",
+      Fp[Banderwagon].fromHex"0x07aa611dcedcb15b720543223d96f0ac8d28fb6cae51baff7ef262ffa8f8c918",
+      Fp[Banderwagon].fromHex"0x15e22b8b7d02f5cf2f49c52c6190369d3ba63f546586f6d2da37c449f8b60f52",
+      Fp[Banderwagon].fromHex"0x20e9cd3a7fca77e38a490835c612d67951d460a1dbfcd267df2b12e5b0330148",
+      Fp[Banderwagon].fromHex"0x713a9164dbe6739dc4f0a81758d181ac9cc72871c1151445d010a9edb4b638a4",
+      Fp[Banderwagon].fromHex"0x18780c47d9f8e63309b97ddd04dcc90920b2b183f7ec0b194222925245f4e2fb",
+      Fp[Banderwagon].fromHex"0x27b9f6421e7859bbb5c8af93f751a2bcf244f49758f2e4f90e5070b9bada029b",
+      Fp[Banderwagon].fromHex"0x0596a3d0ddae097ebcf95c6ee2bfcfce81a2bee506858fc6870b11e11ba54d0f",
+      Fp[Banderwagon].fromHex"0x53aa7d875f42d925180ea84a5115cfd26212655fec382b488ab3113c021da780",
+      Fp[Banderwagon].fromHex"0x463690b59abcf615a6231d154bcbeebf11f69b4a47af75655a9a50847f96ec43",
+      Fp[Banderwagon].fromHex"0x18fdcd6eb725f684d2d28fd9dc0d920b64560ac1281b27b43e0664b6a44982e0",
+      Fp[Banderwagon].fromHex"0x3b25b475ab91194b687a73c92f188612fc010d53ccb225425e544cdf4c887948",
+      Fp[Banderwagon].fromHex"0x67dc39c63dcea51c6dfda5ce9bf2fc0c490a9a547af7f7203c1d97dd9423b5d1",
+      Fp[Banderwagon].fromHex"0x02a8fa6a84eec93cd9bbb5adebfdc2994f75c4fd31a9bdfbb14b44a23cfe6f92",
+      Fp[Banderwagon].fromHex"0x3dbe994d9fc0040d8e1dd927f6f6bf4ac2b901d9b44bf6cee9dd22a497765719",
+      Fp[Banderwagon].fromHex"0x10952f0122e8072dabac70f83e4acd519bbabf1e9e172de6223c4fc056e43d9a",
+      Fp[Banderwagon].fromHex"0x73432a235b185c763e293867324c9a71d98a9a8b48559c156358cf18763f4063",
+      Fp[Banderwagon].fromHex"0x1a112d20409a835fe1b2c3aa0eb455ffbe66f8a0dc0e6bbe91e8a1ba3b6f3441",
+      Fp[Banderwagon].fromHex"0x11c17601029a4196d247f1687a9eb754cbba2b752da27a4e8d66b6b5017225bb",
+      Fp[Banderwagon].fromHex"0x5e73ed8c432405d0ae25b21df6c52b2a6c7876bf0928e68fae34a81e1cfca2ec",
+      Fp[Banderwagon].fromHex"0x6a945980dade66efbcca660ee327eb377a1cb8c070f0ce322e9a4db5f138dab3",
+      Fp[Banderwagon].fromHex"0x1f3834a0f387add7faa1f3804fdb73d008ccffd43f1931b63ff630d3f89c4c2c",
+      Fp[Banderwagon].fromHex"0x1f2caa7aeef7d8dee0eb9d91cdecccfbbf5354ab6e8c2ab4f007bdc150d024b1",
+      Fp[Banderwagon].fromHex"0x640d097461a2eeaf4e84b3cd7dc75b61db872ef3dce28e788d28c143168bba2c",
+      Fp[Banderwagon].fromHex"0x36e1ebe124834a8fcfdc1da7bf59004b42f464cb4256a3b3f3f90bd2b05e602e",
+      Fp[Banderwagon].fromHex"0x60c7efec34093e9a725ce48f79bf9cd60081066e578d78bf6207cd2f20df9d12",
+      Fp[Banderwagon].fromHex"0x18ba7658084c71a467e701a945e9b713df7b4ee66e4caccb7d01866add958780"
+    ]
+  ]
```

Comment:
Note for future improvement, a Sage script generates this table as this can benefit BLS12-377 and the Pasta curves